api/Element.json

```diff
@@ -6487,11 +6487,11 @@
               }
             ],
             "safari": {
-              "version_added": "6",
+              "version_added": "5.1",
               "prefix": "webkit"
             },
             "safari_ios": {
-              "version_added": "6",
+              "version_added": "12",
               "prefix": "webkit",
               "notes": "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled."
```

Comment:
Per https://www.reddit.com/r/apple/comments/b5ketr/psa_ios_122_has_fullscreen_api_custom_html5_video/ it seems like this note only applies since 12.2, but that's fine, no need to capture that fact.